### PR TITLE
add more metrics

### PR DESCRIPTION
**Pull request by Ehco1996**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-dm) that's required for repo owners to accept my contribution.

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v2.0 (TiDB DM 2.0 versions)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/dm/pull/1778/
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [x] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: zh/monitor-a-dm-cluster.md [hunk 1]
```diff
@@ -78,7 +79,8 @@ overview 下包含运行当前选定 task 的所有 DM-worker/master instance/so
 | metric 名称 | 说明  | 告警说明 | 告警级别 |
 |:----|:------------|:----|:----|
 | remaining time to sync | 预计 Syncer 还需要多少分钟可以和 master 完全同步，单位：分钟 | N/A | N/A |
-| replicate lag | master 到 Syncer 的 binlog 复制延迟时间，单位：秒 | N/A | N/A |
+| replicate lag gauge | master 到 Syncer 的 binlog 复制延迟时间，单位：秒 | N/A | N/A |
+| replicate lag histogram | master 到 Syncer 的 binlog 复制延迟时间，单位：秒 由于是 prometheus 内置的直方图类型，数据不一定准确 | N/A | N/A |
```
Comment (lance6716) [inline zh/monitor-a-dm-cluster.md:83]:
```suggestion
| replicate lag histogram | master 到 Syncer 的 binlog 复制延迟分布，单位：秒。注意由于统计机制不同，数据会有误差 | N/A | N/A |
```

File: zh/monitor-a-dm-cluster.md [hunk 2]
```diff
@@ -98,6 +100,11 @@ overview 下包含运行当前选定 task 的所有 DM-worker/master instance/so
 | skipped event duration | binlog replication unit 跳过 binlog event 的耗时，单位：秒 | N/A | N/A |
 | unsynced tables | 当前子任务内还未收到 shard DDL 的分表数量 | N/A | N/A |
 | shard lock resolving | 当前子任务是否正在等待 shard DDL 迁移，大于 0 表示正在等待迁移 | N/A | N/A |
+| ideal qps | 理想情况下可以达到的最高QPS | N/A | N/A |
```
Comment (lance6716) [inline zh/monitor-a-dm-cluster.md:103]:
```suggestion
| ideal qps | 在 DM 运行耗时为 0 时可以达到的最高 QPS | N/A | N/A |
```

**Review by lance6716 — COMMENTED** [review]
rest lgtm

**Review by lance6716 — APPROVED** [review]

**Review by Liuxiaozhen12 — APPROVED** [review]

**Pull request merged**